azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/core/refresktoken/RefreshTokenTenantCredential.java

```diff
@@ -51,19 +57,13 @@ private static AccessToken fromAuthenticationResult(AuthenticationResult authent
         return new AccessToken(authenticationResult.getAccessToken(), expiresOnDate);
     }
 
-    private static AuthenticationResult authorize(AzureEnvironment env,
-                                                  String clientId,
-                                                  String tenantId,
-                                                  String refreshToken,
-                                                  String resource) throws LoginFailureException, MalformedURLException {
+    private AuthenticationResult authenticate(String tenantId, String resource) throws LoginFailureException, MalformedURLException {
```

Comment:
convert to `AccessToken`

Reply:
will do the inner interface update in later PR